_search-plugins/hybrid-search.md

```diff
@@ -569,4 +569,454 @@ The response contains the matching documents and the aggregation results:
     }
   }
 }
+```
+
+## Hybrid search with sorting
+**Introduced 2.16**
+{: .label .label-purple }
+
+By default, the standard hybrid search without sorting returns search results that are ordered by scores in decreasing order.
+With the release of this feature, you can now sort the hybrid search results by providing the `sort` criteria in the search request. For more information on the sort criteria, see [sort results](https://opensearch.org/docs/latest/search-plugins/searching-data/sort/).
+Moreover, hybrid search when sorting applied fetches results from the shards on the basis of sort criteria passed in the search request, therefore, the search results will be sorted as per sort criteria and the scores will be `null`. Scores will be present in the hybrid search sorting results if the sort criteria is by `_score`. 
```

Comment:
```suggestion
When sorting is applied to hybrid search, results are fetched from the shards based on the specified sort criteria. As a result, the search results will be sorted accordingly, and the scores will be null. Scores will only be present if the sort criteria include _score.
```

Reply:
In the last sentence you have used 
```
include _score
```
Here we can't use word include because a user cannot pass _score and another field criteria together as this case is blocked for hybrid query.

To be precise I am talking about below case

```
"sort" : [
   {
      "_score": {
            "order":"desc"
      }
   },
   {
      "_id":{
           "order": "desc"
      }
   }
]
```

Therefore, I think below line will convey this message clearly

```
Scores will only be present if the sort criteria is by _score.
```

Reply:
agree